stable/gcloud-sqlproxy/README.md

```diff
@@ -69,6 +69,7 @@ The following table lists the configurable parameters of the `gcloud-sqlproxy` c
 | `usingGCPController`              | enable the use of the GCP Service Account Controller     | `""`                                                                       |
 | `serviceAccountName`              | specify a service account name to use   | `""`                                                                                        |
 | `cloudsql.instances`              | List of PostgreSQL/MySQL instances      | [{instance: `instance`, project: `project`, region: `region`, port: 5432}] must be provided |
+| `cloudsql.private`                | Flag to enable private ip connection    | false |
```

Comment:
Instead of adding this value, an we instead just use `extraArgs` and add `ip_address_types: PRIVATE` ?